Exiled.Events/Patches/Generic/CheckForLurePatch.cs

```diff
@@ -47,7 +53,12 @@ private static void CheckPlayers()
             {
                 if (Scp106Container.CanBeKilled(player))
                 {
-                    player.Hurt(new UniversalDamageHandler(-1f, DeathTranslations.UsedAs106Bait, null));
+                    var ev = new EnteringFemurBreakerEventArgs(player);
```

Comment:
Please do not use 'var'.
You can use `EnteringFemurBreakerEventArgs ev = new(player);` instead.